usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -54,75 +57,92 @@ def signal_handler(signal, frame):
         signal.signal(signal.SIGINT, signal_handler)
         signal.signal(signal.SIGTERM, signal_handler)
 
-        submission_id = options["submission_id"][0]
+        submission_id = options["submission_id"]
 
-        logger.info("Getting submission {} from broker...".format(submission_id))
-        db_cursor.execute("SELECT * FROM submission WHERE submission_id = %s", [submission_id])
+        logger.info(f"Getting submission {submission_id} from Broker...")
+        db_cursor.execute(
+            f"""
+                select
+                    s.submission_id,
+                    (
+                        select  max(updated_at)
+                        from    {get_publish_history_table()}
+                        where   submission_id = s.submission_id
+                    )::timestamptz as published_date,
+                    (
+                        select  max(updated_at)
+                        from    certify_history
```

Comment:
Staying as `certify_history` in broker, or will they be making a `publish_history` table?

Reply:
I think this may need to use your `get_publish_history_table()` shim, no?

Reply:
There is a `publish_history` and a `certify_history`.  We are pulling `certified_date` from `certify_history`.  We are pulling `published_date` from `certify_history` until `publish_history` becomes available.  Today certification and publication happen at the same time.